recipes/crispector/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: crispector
+  version: "1.0.2b4"
+
+build:
+  number: 0
+  entry_points:
+    - crispector = crispector.cli:main
+  noarch: generic
+  script: python -m pip install . --no-deps -vv
+
+
+source:
+  url: https://github.com/YakhiniGroup/crispector/archive/v1.0.2b4.tar.gz
+  sha256: c074d02083818a088b12ac958db97b89d6d74fea6209b674e2f5a016b326113f
+
+requirements:
+  host:
+    - python 3.7
+    - pip
+    - setuptools
+  run:
+    - python 3.7
+    - biopython >=1.74
+    - fastp
+    - python-edlib
+    - click >=7.0
+    - pyyaml >=5.1.2
+    - matplotlib-base >=3.1.2
+    - scipy >=1.2.1
+    - numpy >=1.12.1
+    - seaborn >=0.9.0
+    - plotly >=4.3.0
+    - pandas >=0.24.2
+    - statsmodels
+    - jinja2
+
+test:
+  commands:
+    - crispector -h
+
+about:
+  home: https://github.com/YakhiniGroup/crispector
+  license: Partners
+  license_file: LICENSE
```

Comment:
The license [explicitly forbids re-distribution](https://github.com/YakhiniGroup/crispector/blob/275fd375cb05b44670ee27d4f44a6b0426ff32a3/LICENSE#L23):
> ... You may not copy or distribute CRISPECTOR and its code in any form.

Are you the copyright holder for the tool? If so, you are of course not bound by those license terms and can just grant Bioconda permission to distribute the tool. To do this properly, I suggest that you
- add a comment to the PR stating that you grant permission to distribute Crispector,
- add a comment to the recipe that links to the PR comment, something like: "permission to distribute the software has been granted by the author at (URL)",
- and optionally, to make it a bit clearer that you’re responsible for the recipe, add yourself as maintainer by adding this section to the recipe (at the bottom):
```
extra:
  recipe-maintainers:
    - iamit87
```
Alternatively, you could of course switch to an Open Source license :-).

If you’re not the copyright holder (or whatever the equivalent in your legislation), you need to get permission first.

Reply:
Hi,
Thanks for the detailed response! 
I'm the author of CRISPECTOR.
 - I added myself as the recipe-maintainers.
 - I added a comment in this PR which grant distribute permissions.
-  I added reference to this PR in the README file.